Sources/DataPipeline/Config/DataPipelineConfigOptions.swift

```diff
@@ -0,0 +1,28 @@
+import CioInternalCommon
+import Foundation
+import Segment
+
+/// Configuration options for the Customer.io Data Pipeline module
+public struct DataPipelineConfigOptions {
+    /// Server key
+    public let writeKey: String
```

Comment:
I think we might need to add `siteId` here too for  migration ! 🤔

Reply:
Thanks for highlighting, I'll add so you can reuse it.